### PR TITLE
Fixed some stuff

### PR DESCRIPTION
**Pull request by xTymonWasTaken**
look in commits

### CODE REVIEW

File: items/OCELOT;4.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   ],
   "internalname": "OCELOT;4",
   "clickcommand": "viewrecipe",
-  "modver": "REL-1.0.0",
+  "modver": "1.2-REL",
   "recipe": {
     "A1": "ENCHANTED_JUNGLE_LOG:64",
     "A2": "ENCHANTED_JUNGLE_LOG:64",
```
Comment (IRONM00N) [inline items/OCELOT;4.json:33]: Fix Recipe

File: items/MIDAS_SWORD.json [hunk 2]
```diff
@@ -19,7 +19,7 @@
     "§7is §c120 §7if the bid was",
     "§7§650,000,000 Coins §7or higher!",
     "",
-    "§7Price paid: §650,040,000 Coins",
+    "§7Price paid: §750,040,000",
```
Comment (IRONM00N) [inline items/MIDAS_SWORD.json:22]: Why 750 mil?

Comment (IRONM00N) [reply items/MIDAS_SWORD.json:22]: Is that the max?

**Comment by IRONM00N** [comment]
Looks Good

**Review by IRONM00N — APPROVED** [review]

File: items/MIDAS_SWORD.json [hunk 3]
```diff
@@ -19,7 +19,7 @@
     "§7is §c120 §7if the bid was",
     "§7§650,000,000 Coins §7or higher!",
     "",
-    "§7Price paid: §650,040,000 Coins",
+    "§7Price paid: §650,040,000",
```
Comment (IRONM00N) [inline items/MIDAS_SWORD.json:22]: Umm what

**Review by Moulberry — APPROVED** [review]

**Pull request merged**